### DISCUSSION TITLE
Staff Roles?

### DISCUSSION THREAD

**Discussion by dsghi** [KelvinTegelaar/CIPP#241] [Q&A] [answered]
I noticed the user roles recently changed, where can I find the list of options? Also, I see documentation location is in flux, and none of the old links are working at the moment.

**Comment by dsghi** [comment, marked as answer]
NVM, found them it in the config file: https://github.com/KelvinTegelaar/CIPP/blob/master/staticwebapp.config.json

**Comment by KelvinTegelaar** [comment]
The documentation platform was really *just* changes an hour ago. check out the docs here: https://kelvintegelaar.github.io/CIPP/
